sql/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -212,18 +210,9 @@ protected void doStop() {
                 serverChannels = null;
             }
         }
-
         if (bootstrap != null) {
-            bootstrap.shutdown();
-            workerExecutor.shutdown();
-            bossExecutor.shutdown();
-
-            try {
-                workerExecutor.awaitTermination(2, TimeUnit.SECONDS);
-                bossExecutor.awaitTermination(2, TimeUnit.SECONDS);
-            } catch (InterruptedException e) {
-                Thread.currentThread().interrupt();
-            }
+            bootstrap.releaseExternalResources();
```

Comment:
As discussed: I'll leave it at releaseExternalResources because it's documented to release external resources and also call into shutdown. Shutdown is documented to clear internal resources. (Though in the used implementation it doesn't matter which one is called)